.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,74 @@
+name: Bug Report
+description: File a bug report
+title: "[Bug]: "
+labels: ["bug"]
+
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report!
+
+  - type: input
+    id: version
+    attributes:
+      label: Version
+      description: What version of Morpheus are you running?
+      placeholder: "example: 22.09"
+    validations:
+      required: true
+
+  - type: dropdown
+    id: installation-method
+    attributes:
+      label: Which installation methods does this occur on?
+      multiple: true
+      options:
+        - Docker
+        - Conda
+        - Source
+
+  - type: textarea
+    id: what-happened
+    attributes:
+      label: What happened? Describe the bug.
+      description: Also tell us, what did you expect to happen?
+      placeholder: XYZ occured, I expected QRS results
```

Comment:
Can users put markdown here? Can we put markdown in the placeholder? If so, we should put headers for the description and expected behavior